qemu-test-init

```diff
@@ -109,10 +109,7 @@ $BUILD_DIR/test/fakerand
 # grub env
 mkdir -p /tmp/boot/grub
 mount --bind /tmp/boot /boot
-grub-editenv test/grubenv.test create
-touch /tmp/boot/grub/grubenv
-mount --bind test/grubenv.test /tmp/boot/grub/grubenv
-
+grub-editenv /tmp/boot/grub/grubenv create
```

Comment:
In the next patch, `grub-editenv` seems to be enough. Do we need to use the `create` command here? If not, just drop it.

Reply:
I don't think create is required. Will drop it.